### PR TITLE
feat: Added console UI utlities

### PR DESCRIPTION
**Pull request by Danwakeem**
## Description
Moving some of the common functions into this utilities library so that we can use them in the CLI and in the web console 😎 

These components will fist be used in serverless/serverless#11782

### CODE REVIEW

File: console-ui.js [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+/**
+ * This function takes in a console span, specifically an aws sdk span, and returns a formatted name
+ *
+ * @param {object} activity A console span object
+ * @returns {string} The formatted span name
+ */
+const formatAWSSDKName = (activity) => {
+  const { name } = activity;
+  const [, , service, operation] = name.split('.');
+  let finalName = `AWS SDK • ${service.toUpperCase()} • ${operation.toUpperCase()}`;
```
Comment (medikoo) [inline console-ui.js:24]: I think this should be in final else clause

File: console-ui.js [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+ * @param {object} activity This is a console event object
+ * @returns {{ message: string, payload: object }} The formatted event where the message can be used as a title and the payload is the JSON combination of the customTags and warning or error attributes
+ */
+const formatConsoleEvent = (activity) => {
+  let message = 'Captured Event';
+  let payload = {};
+  let customTags = {};
+  try {
+    const parsedTags = JSON.parse(activity.customTags);
+    customTags = Object.keys(parsedTags).length > 0 ? { customTags: parsedTags } : {};
```
Comment (medikoo) [inline console-ui.js:74]: I think it'll be clenear to not wrap `customTags` into `{ customTags }` (having that what's assigned to `customTags` is more a `customTagsContainer`, and things get confusing).

Instead, I'll probably keep `customTags` as `undefined` or _tags collection_ and below assign it to payload naturally as:

```javascript
payload = {
  customTags
}
```

(note that `undefined` properties are naturally stripped on JSON, but if having `null` literally assigned is not harmful, it's nicer to use `null` in code)

File: console-ui.js [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+  } catch {
+    // ignore
```
Comment (medikoo) [inline console-ui.js:76]: Do we envision the case of `activity.customTags` being a value but not a JSON string? What is the case for that?

**Review by medikoo — COMMENTED** [review]
@Danwakeem that looks very neat. I've proposed just a few minor improvements

File: console-ui.js (hunk 3 above)
Comment (Danwakeem) [reply console-ui.js:76]: The backend stores customTags as a string so by the time it runs through ingest it gets converted to something like `"{}"`. It is a bit annoying TBH but it is just so the backend doesn't have to handle the data differently per data target.

File: console-ui.js (hunk 1 above)
Comment (Danwakeem) [reply console-ui.js]: Good call 👍

File: console-ui.js (hunk 3 above)
Comment (medikoo) [reply console-ui.js:76]: Ok, but `{}` is a valid JSON string. So we don't have a case when it's not a JSON string, right?

Comment (medikoo) [reply console-ui.js:76]: Test

Comment (medikoo) [reply console-ui.js:76]: Yes, if there's no confirmed reason for try/catch I'd remove it.

Comment (Danwakeem) [reply console-ui.js]: Yeah good call, I went ahead and removed it 😎

**Review by medikoo — APPROVED** [review]
Looks great 👍

**Pull request merged**